Sloth.Core/Domain/JobRecord.cs

```diff
@@ -1,20 +1,51 @@
 using System;
-using System.Collections.Generic;
 using System.ComponentModel.DataAnnotations;
+using Newtonsoft.Json;
```

Comment:
We are still using Newronsoft?

Reply:
I think so in sloth but i can double check

Reply:
I changed it to use system.text.